src/index.html

```diff
@@ -5,9 +5,51 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div class="card" data-qa="card">
+      <div class="card__product-image-cnt">
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+        >
+      </div>
+
+      <div class="card__content">
+        <h3 class="card__product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <div class="card__product-code">
```

Comment:
```suggestion
        <p class="card__product-code">
```
Consider using semantic tags for the text